services/repository/check.go

```diff
@@ -73,7 +73,9 @@ func GitGcRepos(ctx context.Context, timeout time.Duration, args ...git.CmdArg)
 				return db.ErrCancelledf("before GC of %s", repo.FullName())
 			default:
 			}
-			return GitGcRepo(ctx, repo, timeout, args)
+			// we can ignore the error here because it will be logged in GitGCRepo
+			_ = GitGcRepo(ctx, repo, timeout, args)
```

Comment:
```suggestion
			GitGcRepo(ctx, repo, timeout, args)
```
should be possible too, right?

Reply:
We may have a litter that complains (not sure), plus for me I like mentally noting that an error is explicitly ignored here.